app.js

```diff
@@ -1,10 +1,53 @@
 const WebSocket = require('ws');
 require('dotenv').load();
-var exp = require('./index.js');
 
+// when socket opens, notify channel
+function onOpen (soc) {
+    console.log('socket opened');
+    var msg = {
+        "id": 1,
+        "type": "message",
+        "channel": "C1BBWJ7PF",
+        "text": "WhoopBot connected to WebSocket"
+    };
+    
+    if (soc.readyState == 1) {
+        soc.send(JSON.stringify(msg));
+    };
```

Comment:
`readyState` is indeed a number, it's a mapping to a number of states like `OPEN`, `CLOSING`, `CLOSED` ect represented as numbers